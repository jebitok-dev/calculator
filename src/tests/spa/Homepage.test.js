import renderer from 'react-test-renderer';
import React from 'react';
import Home from '../../components/Homepage';

it('should match the Homepage snapshot', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
