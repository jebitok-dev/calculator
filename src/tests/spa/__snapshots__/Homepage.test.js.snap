// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the Homepage snapshot 1`] = `
<div
  className="centered"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero vel quibusdam veritatis quaerat excepturi exercitationem numquam eligendi ut, mollitia voluptate! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Recusandae nihil praesentium reiciendis ut fugiat. Voluptas rem natus aut magni assumenda. Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem! Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti mollitia ullam voluptatem quam exercitationem!
  </p>
</div>
`;
